_posts/2016-12-04-contributing-blog-post-to-pulpproject.org.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Contributing a Blog Post to pulpproject.org
+author: Dennis Kliban
+---
+Quickstart Guide
+----------------
+
+1. Fork the pulpproject.org repository on GitHub.
+   [https://github.com/pulp/pulpproject.org/](https://github.com/pulp/pulpproject.org/)
+
+1. Clone your fork locally and checkout the `gh-pages` branch.
+
+1. Create a new file in the `_posts` directory. The name for the file should be in the following
+   format: `YYYY-MM-DD-title-of-post.md` e.g.
+   `2016-11-30-contributing-blog-post-to-pulpproject.org.md`
+
+1. Write a blog post in the new file.
+
+1. Commit your changes to the `gh-pages` branch.
+
+1. Push your `gh-pages` branch to GitHub.
+
+1. Preview the blog at `https://<username>.github.io/pulpproject.org/blog/`. You can also build the
+   site locally and view your blog post. More information on building the site can be found in the
+   [README](https://github.com/pulp/pulpproject.org/blob/gh-pages/README.md).
+
+1. Create a Pull Request against the `gh-pages` branch of [pulproject.org
+   repository](https://github.com/pulp/pulpproject.org/).
+
+Writing the Blog Post
+---------------------
+
+Pulpproject.org is a [GitHub Pages](https://pages.github.com/) site. Each page is created using
+[GitHub flavored Markdown](https://help.github.com/articles/basic-writing-and-formatting-syntax/).
+Each blog post begins with the [YAML Front Matter](http://jekyllrb.com/docs/frontmatter/). Front
+Matter begins and ends with `---`. In between the dashes, the author specifies an `author` and a
+`title`. `tags` can be specified optionally. List of available tags can be found
+[here](https://github.com/pulp/pulpproject.org/tree/gh-pages/tags). Including a tag or [multiple
+tags](https://raw.githubusercontent.com/pulp/pulpproject.org/gh-pages/_posts/2016-11-17-django14-epel6-retirement.md)
+makes the post discoverable by looking at posts organized by tag. More detailed documentation on
+creating blog posts is available [here](http://jekyllrb.com/docs/posts/).
+
+    ---
+    title: Django14 Retirement From EPEL 6
+    author: Brian Bouterse
+    tags:
+      - platform
+      - rpm
+    ---
+
+Previewing the HTML Site
+------------------------
+
+Your fork of the pulpproject.org repository should already be configured to serve the `gh-pages`
+branch at `http://<username>.github.io/pulpproject.org`. Once you push your changes to the
+`gh-pages` branch, the blog can be viewed at `https://<username>.github.io/pulpproject.org/blog/`.
+It may take a few minutes for the pages to be regenerated.
+
+Another option for previewing your blog entry is to build it locally. The simplest way to prepare
```

Comment:
Can a section be added for this part called `Previewing the HTML Site Locally`

Alternatively we could make the `Using Github` and `Locally` subsections, but I think having separate top level sections is the best.